main.cpp

```diff
@@ -40,76 +65,420 @@ Thing: Car Wash
+    float horizontalInput = 0.0f;
+    //2)vertical input
+    float verticalInput = 0.0f;
+    //3)horizontal gain control
+    int horizontalGain = 0;
+    //4)vertical gain control
+    int verticalGain = 0;
+    //5)horizontal sweep control
+    int horizontalSweepControl = 0;
+//3 things it can do:
+    //1)graph voltage over time
+    void graphVoltageOverTime(float yInput, float sweep);
+    //2)graph the difference in electrical waveforms
+    void graphWaveformDifference (int channelA, int channelB);
+    //3)measure voltage
+    void measureVoltage (int channelA, int channelB);
+
+};
 
 /*
-Thing 2)
+Thing 2)cell phone
 5 properties:
-    1)
-    2)
-    3)
-    4)
-    5)
+    1)carrier
+    2)screen size
+    3)amount of RAM
+    4)whether or not is has a SD card port
+    5)whether or not it has a headphone port
 3 things it can do:
-    1)
-    2)
-    3)
+    1)make calls
+    2)play games
+    3)send emails
  */
+struct CellPhone
+{
+//5 properties:
+    //1)carrier
+    std::string carrier = "Sprint";
+    //2)screen size
+    int screenSize = 3;
+    //3)amount of RAM
+    int gigabytesOfRAM = 8;
+    //4)whether or not is has a SD card port
+    bool hasSDCardPort = false;
+    //5)whether or not it has a headphone port
+    bool phoneHasHeadphoneJack = true;
+   // 5) make 2 of the 10 user-defined types have a nested class. 
+   struct TouchScreen
+    {
+        float screenHeight = 4.5f;
+        float screenWidth = 3.0f;
+        int x = 0, y = 0;
+
+        void fingerPrintVerification(float ownersFinger, float usersFinger);
+        void getFingerPosition(int X, int Y);
+    };
+//3 things it can do:
+    //1)make calls
+    void makeCall (int phoneNumber, std::string personYouAreCalling);
+    //2)play games
+    void playGame (bool gameMode);
+    //3)send emails
+    void sendEmail (std::string emailAddress);
 
+    TouchScreen touchScreen;
+};
 /*
-Thing 3)
+Thing 3)guitar
 5 properties:
-    1)
-    2)
-    3)
-    4)
-    5)
+    1)wood type
+    2)number of frets
+    3)scale length
+    4)whether or not it has a tremelo bridge
+    5)number of strings
 3 things it can do:
-    1)
-    2)
-    3)
+    1)tune the strings
+    2)play a melody
+    3)strum a chord
  */
+ struct Guitar
+ {
+    //5 properties:
+    //1)wood type
+    std::string woodType = "mahogany";
+    //2)number of frets
+    int numberOfFrets = 22;
+    //3)scale length
+    float scaleLength = 24.75f;
+    //4)whether or not it has a tremelo bridge
+    bool hasTremeloBridge = true;
+    //5)number of strings
+    int numberOfstrings = 6;
+   //5) make 2 of the 10 user-defined types have a nested class. 
+    struct Tremelo
+    {
+        int numberOfSprings = 3;
+        bool isTremeloFloating = true;
+        bool isTremeloLocking = true;
+
+        void Flutter(float startFrequency, float endFrequency, float flutterRate);
+        void diveBomb(float startFrequency, float endFrequency, float pitchDecentTime);
+    
+    };
```

Comment:
```
5) make 2 of the 10 user-defined types have a nested class.  
    - this nested class also needs 5 properties and 3 actions.
    - these nested classes are not considered one of your 10 UDTs.
    - this nested class must be related to the class it is nested inside
```
`    - this nested class also needs 5 properties and 3 actions.`

The names of member variables and member functions need to start with a lowercase letter.
- [ ] revise